cluster/gce/util.sh

```diff
@@ -836,6 +867,18 @@ function delete-network() {
 
 function delete-subnetworks() {
   if [[ ${ENABLE_IP_ALIASES:-} != "true" ]]; then
+    if [[ "${ENABLE_BIG_CLUSTER_SUBNETS}" = "true" ]]; then
```

Comment:
Should we delete the default subnetworks? As I understand it, you expand the existing subnetwork, but then we delete them here. Should we keep the default (but expanded) subnetwork? Why is deletion required in that case?